docs/src/appendix/forge-library/parse_json.md

```diff
@@ -0,0 +1,82 @@
+# `parse_json`
+
+
+> `trait FileParser<T, impl TSerde: Serde<T>> {
+>fn parse_json(file: @File) -> Option<T>;
+> }`
+
+Parses json file content and tries to deserialize it to type `T` that implements `Serde` trait.
+
+- `file` - a snapshot of an instance of the struct `File` that consists of the following fields:
+    - `path` - Cairo shortstring representing a path to a file relative to a package root.
+
+> ⚠️ **Warning**
+>
+>  JSON object is an unordered data. To give it an order, the values in the array are sorted alphabetically by JSON key. That means that in order to decode the JSON object correctly, you will need to define attributes of the struct with types that correspond to the values of the alphabetical order of the keys of the JSON.
```

Comment:
The last sentence is too long and I don't really get the meaning of this part:
"to define attributes of the struct with types that correspond to the values of the alphabetical order of the keys of the JSON"